style/pileup.css

```diff
@@ -119,7 +124,7 @@
 /* pileup track */
 .pileup-root > .pileup {
   flex: 1;  /* stretch to fill remaining space */
-  overflow-y: auto;
+  border-top: 1px solid #666;
```

Comment:
I wonder if we should add this to all tracks? Putting a 1px border on top of pileup tracks is specific to the arrangement that we're using on the demo page, but it might not make sense if the pileup track were on top, say, and the other tracks were underneath it.